spec/dummy/config/routes.rb

```diff
@@ -19,4 +19,5 @@
   get "error-summary", to: "welcome#errorsummary"
   get "tabsexample", to: "welcome#tabsexample"
   get "table", to: "welcome#table"
+  get "/api/personalisation/check-email-subscription", to: proc { [404, {}, [""]] }
```

Comment:
Probably worth putting a comment above this line to explain why it's here and which component it refers to, something like

```
we fake this URL to prevent the Single Page notification button from causing an error in the component guide
```

Reply:
good point